audbackend/core/interface/unversioned.py

```diff
@@ -37,6 +37,42 @@ def checksum(
         """
         return self.backend.checksum(path)
 
+    def copy_file(
+            self,
+            src_path: str,
+            dst_path: str,
+            *,
+            verbose: bool = False,
+    ):
+        r"""Copy file on backend.
+
+        If ``dst_path`` exists
+        with a different checksum,
+        it is overwritten,
+        or otherwise,
+        the operation is silently skipped.
```

Comment:
As above:

```suggestion
        If ``dst_path`` exists
        and has a different checksum,
        it is overwritten.
        Otherwise,
        the operation is silently skipped.
```

Reply:
BTW, what happens if a sub path exists on the backend with the same name as the file. I guess trying to overwrite it will raise an error?
Would be nice to add a test for this case.